### ISSUE TITLE
Incompatiblity with Tweakeroo >= 1.21-0.21.55 (sakura's fork)

### ISSUE THREAD

**Issue by MrAn0nym** [Fallen-Breath/tweakermore#88]
### Bug description

Game fails to apply mixin because a method was moved. Therefore i can't launch, crashing on startup.

### Steps to reproduce

Try launching the game with Tweakeroo v0.21.55 (which is distributed on the official modrinth page, even though it's released in sakura's fork) and TweakerMore v3.23.0

### Relevant logs

<details><summary>Relevant stacktrace</summary>
<p>

```
[14:20:42] [Render thread/ERROR]: Mixin apply for mod tweakermore failed tweakermore.mixins.json:tweaks.features.schematicProPlace.PlacementTweaksMixin from mod tweakermore -> fi.dy.masa.tweakeroo.tweaks.PlacementTweaks: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException @Shadow method isFacingValidFor in tweakermore.mixins.json:tweaks.features.schematicProPlace.PlacementTweaksMixin from mod tweakermore was not located in the target class fi.dy.masa.tweakeroo.tweaks.PlacementTweaks. Using refmap tweakermore-refmap.json
org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow method isFacingValidFor in tweakermore.mixins.json:tweaks.features.schematicProPlace.PlacementTweaksMixin from mod tweakermore was not located in the target class fi.dy.masa.tweakeroo.tweaks.PlacementTweaks. Using refmap tweakermore-refmap.json
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachSpecialMethod(MixinPreProcessorStandard.java:436) ~[sponge-mixin-0.15.4+mixin.0.8.7.jar:0.15.4+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachShadowMethod(MixinPreProcessorStandard.java:412) ~[sponge-mixin-0.15.4+mixin.0.8.7.jar:0.15.4+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attachMethods(MixinPreProcessorStandard.java:340) ~[sponge-mixin-0.15.4+mixin.0.8.7.jar:0.15.4+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.attach(MixinPreProcessorStandard.java:299) ~[sponge-mixin-0.15.4+mixin.0.8.7.jar:0.15.4+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinPreProcessorStandard.createContextFor(MixinPreProcessorStandard.java:277) ~[sponge-mixin-0.15.4+mixin.0.8.7.jar:0.15.4+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinInfo.createContextFor(MixinInfo.java:1292) ~[sponge-mixin-0.15.4+mixin.0.8.7.jar:0.15.4+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:203) ~[sponge-mixin-0.15.4+mixin.0.8.7.jar:0.15.4+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:437) ~[sponge-mixin-0.15.4+mixin.0.8.7.jar:0.15.4+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:418) ~[sponge-mixin-0.15.4+mixin.0.8.7.jar:0.15.4+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) [sponge-mixin-0.15.4+mixin.0.8.7.jar:0.15.4+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) [sponge-mixin-0.15.4+mixin.0.8.7.jar:0.15.4+mixin.0.8.7]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) [sponge-mixin-0.15.4+mixin.0.8.7.jar:0.15.4+mixin.0.8.7]
	at knot/io.gitlab.jfronny.libjf.unsafe.asm.AsmTransformer.transformClassBytes(AsmTransformer.java:75) [libjf-unsafe-v0-3.17.3-b2c8b7bb9bb9f239.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422) [fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) [fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) [fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) [fabric-loader-0.16.9.jar:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) [?:?]
	at knot/fi.dy.masa.tweakeroo.config.Configs.loadFromFile(Configs.java:461) [tweakeroo-fabric-1.21-0.21.55.jar:?]
	at knot/fi.dy.masa.tweakeroo.config.Configs.load(Configs.java:527) [tweakeroo-fabric-1.21-0.21.55.jar:?]
	at knot/fi.dy.masa.malilib.config.ConfigManager.loadAllConfigs(ConfigManager.java:44) [malilib-fabric-1.21-0.21.5.jar:?]
	at knot/fi.dy.masa.malilib.event.InitializationHandler.onGameInitDone(InitializationHandler.java:41) [malilib-fabric-1.21-0.21.5.jar:?]
	at knot/net.minecraft.class_310.handler$eng000$malilib$onInitComplete(class_310.java:29678) [client-intermediary.jar:?]
	at knot/net.minecraft.class_310.<init>(class_310.java:706) [client-intermediary.jar:?]
	at knot/net.minecraft.client.main.Main.main(Main.java:239) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480) [fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.16.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.16.9.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:105) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) [NewLaunch.jar:?]
```

</p>
</details> 

[Complete log](https://mclo.gs/K6doQM5)

### Minecraft version

1.21.1 (most likely newer versions as well, haven't checked though)

### TweakerMore version

3.23.0

### Other information

Relevant commit: https://github.com/sakura-ryoko/tweakeroo/commit/a95589ff459ef2b35fdb567cc1f5409af62d8e70

### Check list

- [X] I have verified that the issue persists in the latest version of the mod.
- [X] I have searched the existing issues and confirmed that this is not a duplicate.

**Comment by Gholts** [comment]
same issue
[crash-2024-12-26_16.24.37-client.txt](https://github.com/user-attachments/files/18251179/crash-2024-12-26_16.24.37-client.txt)

**Comment by TOYBOXSM** [comment]
[crash-2024-12-29_12.33.20-client.txt](https://github.com/user-attachments/files/18268510/crash-2024-12-29_12.33.20-client.txt)
mixin无法应用于tweakermore

**Comment by Fallen-Breath** [comment]
Fixed in v3.23.1
